### ISSUE TITLE
Add Immediate Children Pages Discovery Feature

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#16]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
ZORALab's Bissetii has the capability of detecting immediate children pages autoamtically that enables us to build list pages without needing to manually tabulate each of them. Let's add the same feature into hestiaHUGO.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
HestiaHUGO offers children pages navigation in an auto-discovery manner.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
HestiaHUGO does not auto-discover children pages.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 4dfa77c04f6c3759856a9dc2beee6c0dd25eae5a

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
